src/components/Profile/ProfileImage/ProfileImage.tsx

```diff
@@ -0,0 +1,20 @@
+import "./ProfileImage.css";
+import ProfilePicture from "../../../assets/profile.png";

```

Comment:
This doesn't fulfil the requirement to accept various formats, but we can tackle this in another PR if you want.